script/plugin_tools.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ev
+
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  export FLUTTER_CHANGED_PACKAGES=""
```

Comment:
Can you remove it? I don't think we should be polluting the global namespace if we don't have to.